server/controllers/client.ts

```diff
@@ -22,6 +22,7 @@ clientsRouter.use('/videos/watch/:id',
 
 clientsRouter.use('' +
   '/videos/embed',
+  embedCSP,
```

Comment:
It should be already done with https://github.com/Chocobozzz/PeerTube/pull/1252/files#diff-5c092da67283b0b787b618509fa7b96fR58

Reply:
`embedCSP` is purposedly different from `baseCSP`.

Reply:
oups :+1: